### ISSUE TITLE
Got error when running the executable script pcapToJSON.js

### ISSUE THREAD

**Issue by user1020** [joeferner/node-tshark#1]
Thanks for writing this module. It looks very interesting.

Any idea why?  Thanks in advance.

```
jin1@jin1-HP:~/projs/nodeSrv/node_modules/tshark/bin$ ./pcapToJSON.js -in ~/Downloads/misc.pcap -out -
uncaughtException TypeError: Object true has no method 'pause'
    at new Parser (/home/jin1/projs/nodeSrv/node_modules/pcap-parser/lib/pcap-parser.js:136:15)
    at Object.exports.parse (/home/jin1/projs/nodeSrv/node_modules/pcap-parser/lib/pcap-parser.js:150:10)
    at PcapConverter.convertStream (/home/jin1/projs/nodeSrv/node_modules/tshark/lib/pcapConverter.js:144:22)
    at Object.<anonymous> (/home/jin1/projs/nodeSrv/node_modules/tshark/bin/pcapToJSON.js:55:15)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
```

**Comment by joeferner** [comment]
Which version of node are you running?

**Comment by user1020** [comment]
Sorry, should have given it in the initial report

```
node -v
v0.10.26
```

**Comment by joeferner** [comment]
Try `./pcapToJSON.js --in=~/Downloads/misc.pcap --out=-` (notice the double dashes and equals sign)

Probably need to do better command line parsing

**Comment by user1020** [comment]
Thanks. It helped a bit.  I got the following error though:

```
jin1@jin1-HP:~/projs/nodeSrv/node_modules/tshark/bin$ ./pcapToJSON.js --in=/home/jin1/Downloads/misc.pcap --out=-
tshark error:  tshark: Configuration Profile "node-tshark" does not exist


{}
end
```

I have just installed tshark module:

```
jin1@jin1-HP:~/projs/nodeSrv$ npm ls tshark
/home/jin1/projs/nodeSrv
└── tshark@0.0.1  (git://github.com/joeferner/node-tshark.git#2295c2b75f34db4b11f0ebeb8dcb032d28eefd08)
```

**Comment by joeferner** [comment]
I've added instructions on the readme. You basically need to copy the wireshark profile to your wireshark profile directory.

**Comment by user1020** [comment]
Tried to follow your instruction, but can't find the "node-tshark" directory to copy from

```
jin1@jin1-HP:~/projs/nodeSrv/node_modules/tshark$ ls
bin  index.js  lib  node_modules  package.json  README.md  test  testData
jin1@jin1-HP:~/projs/nodeSrv/node_modules/tshark$ find . -name node-tshark
jin1@jin1-HP:~/projs/nodeSrv/node_modules/tshark$
```

**Comment by joeferner** [comment]
Sorry I haven't published it to npm yet.  Could you just grab the file from here... https://github.com/joeferner/node-tshark/tree/master/wireshark-profiles 

It needs to go in a directory ~/.wireshark/profiles/node-tshark/

If we can get you up and running I'll publish a new version.

**Comment by user1020** [comment]
Cool. It works. Thanks Joe!

**Comment by joeferner** [comment]
Published v0.0.2 with the new profiles and readme.
